cmd/generator/main.go

```diff
@@ -142,12 +145,17 @@ var generatorCmd = &cobra.Command{
 func getReleasePlugins(ctx context.Context, client *github.Client, repositoryName string, includePreRelease bool, existingPlugins []*model.Plugin) ([]*model.Plugin, error) {
 	logger := logger.WithField("repository", repositoryName)
 
-	repository, _, err := client.Repositories.Get(ctx, "mattermost", repositoryName)
+	org := os.Getenv("GITHUB_ORG")
+	if org == "" {
+		org = defaultGitHubOrg
+	}
```

Comment:
Is there a need to support a different org right now? If so, separate PR?

Reply:
@hanzei I used this to test with my own GitHub org, because I thought I needed a GitHub access token for the mattermost org. But I now realize that you just need any GitHub access token so you can query GitHub in general.

Reply:
I'm not against make the org configurable. But if you add this feature to this PR, we have to ensure it's available everywhere. Maybe even use a flag per command?

Reply:
To decrease scope of this change, I will take the configurability out. 0/5 I'm not even sure it's worth making configurable in the first place for the purposes of the marketplace.